tmp/crosshair.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+cd examples
+cd diffbehavior
+cd tool
+crosshair cover a.List > /Users/ShMac/Desktop/CS230-TIM-Improves-Merging/tmp/crosshair-results.out 
+cd /Users/ShMac/Desktop/CS230-TIM-Improves-Merging
+cd examples
+cd diffbehavior
+cd tool
+crosshair cover a.increment_list > /Users/ShMac/Desktop/CS230-TIM-Improves-Merging/tmp/crosshair-results.out 
+cd /Users/ShMac/Desktop/CS230-TIM-Improves-Merging
```

Comment:
ShMac may be an issue

Reply:
this is a generated file so it will get overridden